TransformationSystem/Client/TransformationCLI.py

```diff
@@ -14,6 +14,7 @@
 
 from DIRAC.TransformationSystem.Client.Transformation           import Transformation
 from DIRAC.TransformationSystem.Client.TransformationClient     import TransformationClient
+from DIRAC.Resources.Catalog.FileCatalogClient                  import FileCatalogClient
```

Comment:
You have to redo this PR please. By procedure I can not pick up new changes from a commit in some you branch. So, create a branch which contains the necessary changes with respect to the main rel-v6r12 branch. Push this branch to your forked repository in GitHub. Make a new PR against the rel-v6r12 branch.